flax/linen/transforms.py

```diff
@@ -672,34 +732,57 @@ def __call__(self, c, xs):
 
     assert out_val.shape == (batch_size, seq_len, out_feat)
 
+
+  Note that when providing a function to ``nn.scan``, the scanning happens over
```

Comment:
this is not correct: scanning happens over all argumenst starting from the third

Reply:
Ah yes of course, updated it.

Reply:
Wait, actually I am not sure about your answer. What if you provide `in_axes=0` as in this example? Won't you then scan only over the third argument? I thought if you want to scan over all axes from the third you have to provide `in_axes=(nn.broadcast, 0)`, or is that incorrect?

Reply:
in_axes=0 should apply from the third arg onwards

Reply:
Ah right, now I get it. Updated the docstring.